openattribute.php

```diff
@@ -448,14 +447,34 @@ function openattribute_register() {
 	add_option( 'openattribute_site_attribution_url', '' );
 	add_option( 'openattribute_append_content', '1' );
 	add_option( 'openattribute_append_footer', '1' );
-	add_option( 'openattribute_pre_license_html', '<div>' );
-	add_option( 'openattribute_post_license_html', '</div>' );
+	add_site_option( 'openattribute_pre_license_html', '<div id="open-attribute">' );
+	add_site_option( 'openattribute_post_license_html', '</div>' );
 	add_option( 'openattribute_authoroverride', '' );
 	add_option( 'openattribute_index', 0 );
 	add_option( 'openattribute_indexsingle', 0 );
 	add_option( 'openattribute_postsonly', 0 );
 	add_option( 'openattribute_postsonly', 0 );
+}
 
+function openattribute_network_options_page() {
+    ?>
+    <div class="wrap">
+        <h2>Open Attribute</h2>
+        <p>This control panel allows you to set site-wide options for the Open Attribute plugin.</p>
+        <p>For individual site settings, please visit the specific site's dashboard settings.</p>
+        <form method="post" action="<?php echo $_SERVER['REQUEST_URI']; ?>">
+            <?php
+            openattribute_show_html_options();
+            openattribute_show_add_licenses_options();
+            ?>
+            <p class="submit">
```

Comment:
You need to add a hidden 'submitted' input to this page, so that the save method will know to try and process the form submission.

e.g. <input name="submitted" type="hidden" value="openattribute" />

Reply:
Wow. Thanks for spotting this Richard. Sorry I missed this. Fixed.